integration-tests/src/test/resources/oke/terraform/cluster.tf

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Can we re use the tf files in weblogic-kubernetes-operator/kubernetes/samples/scripts/terraform directory, so that we can verify what we ship with the sample.

Reply:
there are changes applicable to Jenkins run requirements , for example we use only 2 nodes instead 3, also vcn setup is different, we also have extra *tf files for mount target, exports, and file system